### PR TITLE
Disable Timings for 1.19.4+

### LINKED ISSUE #5726: Creating timing - this is deprecated behavior Paper issue
### Skript/Server Version

```markdown
[INFO] [Skript] Server Version: git-Paper-538 (MC: 1.19.4)
[INFO] [Skript] Skript Version: 2.6.4
[INFO] [Skript] Installed Skript Addons: None
[INFO] [Skript] Installed dependencies:
[INFO] [Skript] - WorldGuard v7.0.7+216b061
```

### Bug Description

An warn from the server when using the `wait` expresssions.
```
Plugin 'Skript' is creating timing '<script>: line <line of the trigger>' - this is deprecated behavior, please report it to the authors: Njol, Mirreski, SkriptLang Team, Contributors
```
So its spamming the console and make it undreadeble (whit all my scripts on, the console is so spammed that its cant be logged), but that not the most imortant, because if a error is here, its the mening of a deeper problem, and that not a good news.
"this is deprecated behavior, **please report it**"
Why im here now.

### Expected Behavior

No error in the console, and `wait` expression compatible with the server.

### Steps to Reproduce

Using one of the last paper version (538), the last stable version of skript (2.6.4) and use the `wait` expressions to get this warn.

### Errors or Screenshots

So, here a script using the `wait` expressions in two differents sections:
```
on load:
	send "will wait" to the console
	wait 1 ticks
	send "has wait" to the console
every 10 second:
	send "every event" to the console
	wait 1 tick
	send "every event 2" to the console
```
What's happening in the console using this script:
```
[09:45:49] [INFO] [Skript] Reloading Test.sk...
[09:45:49] [INFO] will wait
[09:45:49] [INFO] [Skript] Successfully reloaded Test.sk. (3ms)
[09:45:49] [WARN] Plugin 'Skript' is creating timing 'Test.sk: line 1' - this is deprecated behavior, please report it to the authors: Njol, Mirreski, SkriptLang Team, Contributors
[09:45:49] [INFO] has wait
[09:45:59] [INFO] every event
[09:45:59] [WARN] Plugin 'Skript' is creating timing 'Test.sk: line 5' - this is deprecated behavior, please report it to the authors: Njol, Mirreski, SkriptLang Team, Contributors
[09:45:59] [INFO] every event 2
```

### Other

no.

### Agreement

- [x] I have read the guidelines above and affirm I am following them with this report.

### PR DESCRIPTION
**Pull request by sovdeeth**
### Description

Prevents Skript from enabling Timings support for Paper versions 1.19.4 or greater. 
Fixes #5726

---
**Target Minecraft Versions:** 
**Requirements:** 
**Related Issues:** #5726

### CODE REVIEW

**Review by AyhamAl-Ali — APPROVED** [review]

**Review by Moderocky — APPROVED** [review]

**Review by TheLimeGlass — APPROVED** [review]
I believe a suggestion issue should be created to add further support into Spark (Spark is what Paper is transitioning to)

**Comment by sovdeeth** [comment]
> I believe a suggestion issue should be created to add further support into Spark (Spark is what Paper is transitioning to)

We may want to wait for https://github.com/lucko/spark/issues/242 to have, like, any progress made. As of right now, I don't believe there is any api available to add additional metadata like line numbers/scripts to bring Spark timings to parity with what we had with Timings.

**Comment by TheLimeGlass** [comment]
> > I believe a suggestion issue should be created to add further support into Spark (Spark is what Paper is transitioning to)
> 
> We may want to wait for [lucko/spark#242](https://github.com/lucko/spark/issues/242) to have, like, any progress made. As of right now, I don't believe there is any api available to add additional metadata like line numbers/scripts to bring Spark timings to parity with what we had with Timings.

That's why an issue suggestion should be made as a reminder.

File: src/main/java/ch/njol/skript/SkriptConfig.java [hunk 1]
```diff
@@ -196,15 +196,22 @@ public static String formatDate(final long timestamp) {
...
-						Skript.info("Timings support enabled!");
-					SkriptTimings.setEnabled(t); // Config option will be used
-				} else { // Not running Paper
+				if (!Skript.classExists("co.aikar.timings.Timings")) { // Check for Paper server
 					if (t) // Warn the server admin that timings won't work
 						Skript.warning("Timings cannot be enabled! You are running Bukkit/Spigot, but Paper is required.");
 					SkriptTimings.setEnabled(false); // Just to be sure, deactivate timings support completely
+					return;
 				}
+				if (Skript.isRunningMinecraft(1, 19, 4)) { // check for valid paper version (<1.19.4)
```
Comment (APickledWalrus) [inline src/main/java/ch/njol/skript/SkriptConfig.java:205]: Would it be better to check for the deprecation annotation?

Comment (sovdeeth) [reply src/main/java/ch/njol/skript/SkriptConfig.java:205]: It may, but honestly I just didn't know that was possible.

Comment (sovdeeth) [reply src/main/java/ch/njol/skript/SkriptConfig.java:205]: After [discussion](https://discord.com/channels/135877399391764480/828337517471203328/1141856381159362610), decided to keep it as is. Possible danger if, as Pickle says, "other platforms end up using it / keeping it but that seems unlikely"

**Review by APickledWalrus — APPROVED** [review]

**Pull request merged**